pkgs/applications/virtualization/xen/4.15.nix

```diff
@@ -0,0 +1,181 @@
+{ lib, callPackage, fetchurl, fetchpatch, fetchgit
+, ocaml-ng
+, withInternalQemu ? true
+, withInternalTraditionalQemu ? true
+, withInternalSeabios ? true
+, withSeabios ? !withInternalSeabios, seabios ? null
+, withInternalOVMF ? false # FIXME: tricky to build
+, withOVMF ? false, OVMF
+, withLibHVM ? false
+
+# xen
+, python3Packages
+
+# qemu
+, udev, pciutils, xorg, SDL, pixman, acl, glusterfs, spice-protocol, usbredir
+, alsa-lib, glib, python3
+, ... } @ args:
+
+assert withInternalSeabios -> !withSeabios;
+assert withInternalOVMF -> !withOVMF;
+assert !withLibHVM;
```

Comment:
If we can't activate it we can also remove that option.

Reply:
I'm not sure if I can remove the option and still be backwards compatible and keep Xen 4.10 though. Personally I'll gladly remove Xen 4.10 completely, but not sure what that could break for others 😕

Reply:
I think we want to keep Xen 4.10 for now probably but we can safely remove the option if it isn't working at all.